.github/workflows/nightly.yaml

```diff
@@ -118,6 +104,43 @@ jobs:
             TAG=${{ steps.meta.outputs.version }}
             COMMIT=${{ github.sha }}
             REPO_INFO=https://github.com/${{ github.repository }}.git
+
+  redhat-certification-test:
+    environment: 'Docker Push'
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    needs: 
+      # needs the env vars from the previous job and cached binary build
```

Comment:
> needs the env vars from the previous job

I'm not sure I follow. Where is that being used?

Reply:
not the env vars saved to `$GITHUB_ENV` and the docer build cache

Reply:
And this:
```
      - name: Cache Docker layers
        uses: actions/cache@v3
        with:
          path: /tmp/.buildx-cache
          key: ${{ runner.os }}-buildx-${{ github.sha }}
          restore-keys: |
            ${{ runner.os }}-buildx-
```
Should used cached layers too. Binary is a cached layer in that case.

Reply:
Hm so 2 points:

- the binary is being built as part of the docker build hence it will not be "available". It will be cached somewhere in the cached layer that we save in `/tmp/.buildx-cache`, right? I believe we're talking about the same thing though.
- as per Github's docs https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable 

  > You can make an environment variable available to any **subsequent steps in a workflow job** by defining or updating the environment variable and writing this to the GITHUB_ENV environment file. The step that creates or updates the environment variable does not have access to the new value, but all subsequent steps in a job will have access.

  So I'm not sure this is true for the envs

Reply:
You are right in both cases. changing the env thing.